ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -292,6 +292,7 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
                 unit={unit}
                 grid={gridOverrides}
                 tooltipConfig={{ wrapLabels: true }}
+                syncGroup="default-panel-group"
```

Comment:
Are there potential use cases where we want to have multiple groupings on a given page? If so, I think it would be good to think about how `syncGroup` could be configurable in the panel settings instead of hardcoded to a single value. I'm cool with throwing a TODO in there for now and fixing that up in a follow-up though.

Reply:
Yeah definitely! Added the TODO, we'll want to think through dashboard settings as well as per panel overrides, especially important for embed use cases too where consumers are using multiple TimeSeriesChart.PanelComponent instances